resolwe/test_helpers/test_runner.py

```diff
@@ -365,18 +366,22 @@ def run_tests(self, test_labels, **kwargs):
         :param test_labels: Labels of tests to run
         """
         if self.only_changes_to:
-            print("Detecting changed files between {} and HEAD.".format(self.only_changes_to))
-            try:
-                changed_files = subprocess.check_output(['git', 'diff', '--name-only', self.only_changes_to, 'HEAD'])
-                changed_files = changed_files.decode('utf8').strip().split('\n')
-                changed_files = [file for file in changed_files if file.strip()]
-
-                top_level_path = subprocess.check_output(['git', 'rev-parse', '--show-toplevel'])
-                top_level_path = top_level_path.decode('utf8').strip()
-            except subprocess.CalledProcessError:
-                print("ERROR: Git command failed.")
+            # Check if there are changed files. We need to be able to switch between branches
```

Comment:
Can you wrap the comments at the same wrap limit as the surrounding code?
Here it would probably at 100 characters.